hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordListBuilder.java

```diff
@@ -39,6 +39,7 @@
  */
 public class RecordListBuilder {
 
+    public static final String CONFIGURATION_MUST_NOT_BE_NULL = "configuration must not be null";
```

Comment:
```suggestion
    private static final String CONFIGURATION_MUST_NOT_BE_NULL = "configuration must not be null";
```
This would be an odd string to export to multiple places

Reply:
Updated.